modules/styling/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@workday/canvas-kit-styling",
+  "version": "9.1.20",
```

Comment:
i think the latest is 9.1.24

Reply:
The value keeps changing and there isn't a good way to make sure it is always up to date :(